capabilities.json

```diff
@@ -1,538 +1,394 @@
-                        "value": "Cambria Math"
-                    },
-                    {
-                        "displayName": "Candara",
-                        "value": "Candara"
-                    },
-                    {
-                        "displayName": "Comic Sans MS",
-                        "value": "Comic Sans MS"
-                    },
-                    {
-                        "displayName": "Consolas",
-                        "value": "Consolas"
-                    },
-                    {
-                        "displayName": "Constantia",
-                        "value": "Constantia"
-                    },
-                    {
-                        "displayName": "Corbel",
-                        "value": "Corbel"
-                    },
-                    {
-                        "displayName": "Courier New",
-                        "value": "Courier New"
-                    },
-                    {
-                        "displayName": "Georgia",
-                        "value": "Georgia"
-                    },
-                    {
-                        "displayName": "Lucida Sans Unicode",
-                        "value": "Lucida Sans Unicode"
-                    },
-                    {
-                        "displayName": "Segoe (Bold)",
-                        "value": "'Segoe UI Bold', wf_segoe-ui_bold, helvetica, arial, sans-serif"
-                    },
-                    {
-                        "displayName": "Segoe UI",
-                        "value": "'Segoe UI', wf_segoe-ui_normal, helvetica, arial, sans-serif;"
-                    },
-                    {
-                        "displayName": "Segoe UI Light",
-                        "value": "'Segoe UI Light', wf_segoe-ui_light, helvetica, arial, sans-serif;"
-                    },
-                    {
-                        "displayName": "Symbol",
-                        "value": "Symbol"
-                    },
-                    {
-                        "displayName": "Tahoma",
-                        "value": "Tahoma"
-                    },
-                    {
-                        "displayName": "Times New Roman",
-                        "value": "Times New Roman"
-                    },
-                    {
-                        "displayName": "Trebuchet MS",
-                        "value": "Trebuchet MS"
-                    },
-                    {
-                        "displayName": "Verdana",
-                        "value": "Verdana"
-                    },
-                    {
-                        "displayName": "Wingdings",
-                        "value": "Wingdings"
-                    }
-                ]
-            }
-          }
-      }
-    }
-  },
-  "supportsKeyboardFocus": true,
-  "supportsMultiVisualSelection": true
-}
+    "supportsKeyboardFocus": true,
+    "supportsMultiVisualSelection": true
```

Comment:
please keep the existing formatting and modify only the required lines.

Reply:
Done: [e79b3ad](https://github.com/microsoft/powerbi-visuals-heatmap/pull/118/commits/e79b3ada6935206f8a1106748540d26f1ac45d39) and [deb57e4](https://github.com/microsoft/powerbi-visuals-heatmap/pull/118/commits/deb57e41e5a7e1d73f697023dc2268590afe9585)